custom/abdm/milestone_one/utils/response_util.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework.response import Response
+
+from rest_framework.status import HTTP_200_OK, HTTP_500_INTERNAL_SERVER_ERROR, HTTP_400_BAD_REQUEST
+
+success_response_keys = ["txnId", "healthIdNumber", "auth_methods", "token"]
+
+
+def get_response(response_data):
+    if not response_data:
+        return Response({"error": "No valid response found"}, status=HTTP_500_INTERNAL_SERVER_ERROR)
+    if any([key in response_data for key in success_response_keys]):
+        return _get_success_abdm_response(response_data)
+    return _get_error_abdm_response(response_data)
+
+
+def generate_invalid_req_response(message):
+    resp = {
+        "code": "400",
```

Comment:
This format is used by ABDM for error details. So we are returning client errors also along the same lines so that end user(ABDM app here) need not worry about varying response formats.

Reply:
Ya, i just meant we don't need to hardcode 400. I assume `HTTP_400_BAD_REQUEST` would be 400 as well.

Reply:
Got it. There is a difference of type though, string and int. We can typecast `HTTP_400_BAD_REQUEST` to str here in that case. What do you say?

Reply:
ya, we can typecast if needed. I believe an int my work as well.

Reply:
Typecasted to str as this is the format in which ABDM sends response and we have kept ours also inline with it.